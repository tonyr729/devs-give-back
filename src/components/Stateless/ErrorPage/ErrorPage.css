.sorry-text {
  font-size: 2rem;
  padding-top: 10rem; }

.message {
  font-size: 3.5rem;
  padding: 4rem 0 8rem;
  width: 60rem;
  line-height: 5rem; }

a p {
  font-size: 2rem;
  padding: 7rem; }
  a p:hover {
    color: #29bbff; }
