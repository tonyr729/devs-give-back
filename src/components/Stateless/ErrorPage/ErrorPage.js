import React from 'react';
import { NavLink } from 'react-router-dom';
import './ErrorPage.css';

export const ErrorPage = () => {
  return (
    <div>
      <p className='sorry-text'>Awkward</p>
      <p className='message'>Something went wrong on our end. <br /> Sorry about that...</p>
      <NavLink to='/'>
        <p>Go back</p>
      </NavLink>
    </div>
  );
};