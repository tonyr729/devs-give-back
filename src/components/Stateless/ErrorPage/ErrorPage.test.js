import { shallow } from 'enzyme';
import React from 'react';
import { ErrorPage } from './ErrorPage'

describe('ErrorPage', () => {
  it('should match snapshot', () => {
    const errorPage = shallow(<ErrorPage  />)

    expect(errorPage).toMatchSnapshot();
  });
});