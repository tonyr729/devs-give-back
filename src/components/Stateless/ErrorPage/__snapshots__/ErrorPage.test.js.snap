// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorPage should match snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <p
          className="sorry-text"
>
          Awkward
</p>,
        <p
          className="message"
>
          Something went wrong on our end. 
          <br />
           Sorry about that...
</p>,
        <NavLink
          activeClassName="active"
          ariaCurrent="true"
          to="/"
>
          <p>
                    Go back
          </p>
</NavLink>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Awkward",
          "className": "sorry-text",
        },
        "ref": null,
        "rendered": "Awkward",
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            "Something went wrong on our end. ",
            <br />,
            " Sorry about that...",
          ],
          "className": "message",
        },
        "ref": null,
        "rendered": Array [
          "Something went wrong on our end. ",
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": "br",
          },
          " Sorry about that...",
        ],
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "activeClassName": "active",
          "ariaCurrent": "true",
          "children": <p>
            Go back
</p>,
          "to": "/",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Go back",
          },
          "ref": null,
          "rendered": "Go back",
          "type": "p",
        },
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <p
            className="sorry-text"
>
            Awkward
</p>,
          <p
            className="message"
>
            Something went wrong on our end. 
            <br />
             Sorry about that...
</p>,
          <NavLink
            activeClassName="active"
            ariaCurrent="true"
            to="/"
>
            <p>
                        Go back
            </p>
</NavLink>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Awkward",
            "className": "sorry-text",
          },
          "ref": null,
          "rendered": "Awkward",
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "Something went wrong on our end. ",
              <br />,
              " Sorry about that...",
            ],
            "className": "message",
          },
          "ref": null,
          "rendered": Array [
            "Something went wrong on our end. ",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": "br",
            },
            " Sorry about that...",
          ],
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "activeClassName": "active",
            "ariaCurrent": "true",
            "children": <p>
              Go back
</p>,
            "to": "/",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Go back",
            },
            "ref": null,
            "rendered": "Go back",
            "type": "p",
          },
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
